Tests/ContentTypeTests.swift

```diff
@@ -63,7 +63,7 @@ class ContentTypeTests: ContentfulBaseTests {
                         expect(array.items.count).to(equal(4))
 
                         let _ = array.items.first.flatMap { (type: ContentType) in
-                            expect(type.name).to(equal("Cat"))
+                            expect(type.name).to(equal("City"))
```

Comment:
That looks surprising: Did the cat change to a city? Or did the ordering change?